### ISSUE TITLE
Problems using UUID There is no [permission] with ID `85d49adb-e164-41e5-afd2-3e3c45af660a` for guard `web`.

### ISSUE THREAD

**Issue by josecastro22** [spatie/laravel-permission#2737]
### Description

Im using this seeder to create an admin user and assing all roles in database

```
        $user = User::create([
            'email' => 'admin@domain.com',
            'password' => 'password',
        ]);

        $perfil = Perfil::create([
            'user_id' => $user->id,
            'name' => 'Admin',
            'last_name' => 'App',
        ]);

        $role = Role::create(['name' => 'admin']);

        $permissions = Permission::pluck('uuid', 'uuid')->all();

        $role->syncPermissions($permissions);

        $user->assignRole([$role->uuid]);
```

The problem its when execute **$role->syncPermissions($permissions);** i get this error

> There is no [permission] with ID `488a71c3-d01b-45fc-8fee-03422322c4ec` for guard `web`.

I follow the error and come to 

> vendor\spatie\laravel-permission\src\Traits\HasPermissions.php:491

So i encountered with this line

` return $this->getPermissionClass()::findById($permissions, $this->getDefaultGuardName());`

So i try to get the Permission just using 

`Permission::findById('488a71c3-d01b-45fc-8fee-03422322c4ec', 'web')`

And there is not problem. So what its wrong?

### Steps To Reproduce

1.
2.
3.
...

### Version of spatie/laravel-permission package:

^6.9

### Version of laravel/framework package:

^11.9

### PHP version:

^8.2

**Comment by parallels999** [comment]
Seems like cache problem again, there are many similar threads

**Comment by drbyte** [comment]
Yes, could be a cache issue, or perhaps something incomplete with the UUID configuration that's causing `-> getKeyName()` to not return the correct field name.
